inference-engine/src/mkldnn_plugin/nodes/mkldnn_fullyconnected_node.cpp

```diff
@@ -268,12 +269,8 @@ const std::vector<impl_desc_type>& MKLDNNFullyConnectedNode::getPrimitivesPriori
     return implPriorities;
 }
 
-MKLDNNNode::AttrPtr MKLDNNFullyConnectedNode::initPrimitiveAttr() {
-    auto attr = std::make_shared<mkldnn::primitive_attr>(mkldnn::primitive_attr());
-
-    setPostOps(*attr, true, true);
-
-    return attr;
+MKLDNNNode::AttrPtr MKLDNNFullyConnectedNode::initPrimitiveAttr() const {
```

Comment:
The change seems to be invalid. At least because initPrimitiveAttr  can be also called in base MKLDNNNode class. 
That's the error coverity reports here?

Reply:
The invalid change was reverted. This issue (missing override qualifier) will be fixed in #8009.